cogs/giftdrop.py

```diff
@@ -423,22 +431,22 @@ async def change_nickname_command(self, ctx: commands.Context, target: discord.M
                         WHERE user_id = $1
                         RETURNING *
                         """,
-                        target.id, 
+                        target.id,
                         nickname
                     )
                     if ret_value is None:
                         await ctx.send(f"{ctx.author.mention} Sorry, that user has not joined the event yet.")
                     else:
-                        await ctx.send(f"{ctx.author.mention}, The nickname was successfully changed to **{ret_value['nickname']}**!")
+                        await ctx.send(
```

Comment:
I understand this line is a tad long, but there is no real need to split it like this for 11 characters.